packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -4044,9 +4044,9 @@ User permissions are only effective regarding the Groups to which the admin is g
 
 | Permission type                         | Description                                                                                                                                           | Applicable resource types                    |
 | :-------------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------------------- |
-| `okta.users.manage`                     | Allows the admin to create and manage Users and read all profile and credential information for Users                                                | All Users, all Users within a specific Group |
+| `okta.users.manage`                     | Allows the admin to create and manage Users and read all profile and credential information for Users. Delegated admins with this permission can only manage user credential fields and not the credential values themselves. | All Users, all Users within a specific Group |
 | `okta.users.create`                     | Allows the admin to create Users. If the admin is also scoped to manage a Group, that admin can add the User to the Group on creation and then manage.| All Groups, a specific Group                 |
-| `okta.users.read`                       | Allows the admin to read any User's profile and credential information                                                                               | All Users, all Users within a specific Group |
+| `okta.users.read`                       | Allows the admin to read any User's profile and credential information. Delegated admins with this permission can only manage user credential fields and not the credential values themselves. | All Users, all Users within a specific Group |
 | `okta.users.credentials.manage`         | Allows the admin to manage only credential lifecycle operations for a User                                                                           | All Users, all Users within a specific Group |
 | `okta.users.credentials.resetFactors`   | Allows the admin to reset MFA authenticators for users                                                                                               | All Users, all Users within a specific Group |
 | `okta.users.credentials.resetPassword`  | Allows the admin to reset passwords for users                                                                                                        | All Users, all Users within a specific Group |
```

Comment:
```suggestion
| `okta.users.credentials.resetPassword`  | Allows the admin to reset passwords for users                                                                                                        | All users, all users within a specific group |
```

Reply:
There's a bunch of capitalization stuff below this line that I want to fix, but the PR won't let me. Can you please fix? (Group > group, App > app, Users > users, Devices > devices, organization > org, etc.)